web/src/app/users/UserContactMethodVerificationDialog.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import p from 'prop-types'
+import FormDialog from '../dialogs/FormDialog'
+import gql from 'graphql-tag'
+import { Mutation } from 'react-apollo'
+import Query from '../util/Query'
+import { fieldErrors, nonFieldErrors } from '../util/errutil'
+import UserContactMethodVerificationForm from './UserContactMethodVerificationForm'
+import { graphql2Client } from '../apollo'
+
+/*
+ * Reactivates a cm if disabled and the verification code matches
+ */
+const verifyContactMethodMutation = gql`
+  mutation verifyContactMethod($input: VerifyContactMethodInput!) {
+    verifyContactMethod(input: $input)
+  }
+`
+
+/*
+ * Get cm data so this component isn't dependent on parent props
+ */
+const contactMethodQuery = gql`
+  query($id: ID!) {
+    userContactMethod(id: $id) {
+      id
+      name
+      type
+      value
+    }
+  }
+`
+
+/*
+ * Used for the subtitle of the dialog
+ */
+function formatNumber(n) {
```

Comment:
Is there a reason we can't use the existing function `formatPhoneNumber` in `web/src/app/users/util.js` for this?  There also appears to be formatting differences between this new function and existing.

Reply:
There's not! I didn't know we had that function